### ISSUE TITLE
job stop executing when setting def concurrent = false

### ISSUE THREAD

**Issue by ppazos** [grails-plugins/grails-quartz#41]
I'm using grails 2.4.4, quartz plugin 1.0.2

The job is not executing at all.

```
class IndexDataJob {

   def concurrent = false

   static triggers = {
      simple repeatInterval: 500l, startDelay:240000 // execute job once in 45 seconds
   }

   def dataIndexerService

   def execute()
   {
      ...
   }
}
```

**Comment by ppazos** [comment]
I created another job from scratch and it executed ok with the concurrent false, until I added the startDelay:240000

That was the issue. Is this a bug or the expected behavior?

**Comment by davydotcom** [comment]
There was a bug with startDelay if startDelay was null, the scheduler would not always run the job on a simple trigger. Especially when using jdbcStore

**Comment by davydotcom** [comment]
fixed in 2.0.12
